docs/guide.rst

```diff
@@ -11,14 +11,14 @@ Let's say we have a contract with this interface:
 
 .. literalinclude:: ../starknet_py/tests/e2e/docs/guide/test_using_existing_contracts.py
```

Comment:
Shouldn't it be changed to `.. codesnippet`?

Reply:
`literalinclude` can be used the same way as `codesnippet` with the exception that it will fetch only the first part of a file between `:start-after:` and `:end-before:` options